src/components/typography/font-example.tsx

```diff
@@ -0,0 +1,37 @@
+import { Box, Typography } from '@mui/material';
+import * as React from 'react';
+
+export interface ExampleProps {
+  title: fontOption;
+  content: string;
+}
+
+interface FontWeightAndStyle {
+  fontWeight: string;
+  fontStyle?: string;
+}
+
+type fontOption = 'Light Italic' | 'Medium' | 'Regular';
+
+function getFontWeightAndFontStyle(s: fontOption): FontWeightAndStyle {
+  if (s === 'Light Italic') {
+    return { fontWeight: '300', fontStyle: 'italic' };
+  } else if (s === 'Medium') {
+    return { fontWeight: '500' };
+  } else {
+    return { fontWeight: '400' };
+  }
+}
+
+export default function FontExample({ title, content }: ExampleProps) {

```

Comment:
Preferred stylistically:

```
const FontExample: React.FC<ExampleProps> = ({title, content}) => {
    ...
}
export default FontExample;
```

Guarantees  we are typing as a functional react component

Reply:
All has been changed